src/Symfony/Component/Console/ConsoleEvents.php

```diff
@@ -40,6 +41,14 @@ final class ConsoleEvents
      */
     public const SIGNAL = 'console.signal';
 
+    /**
+     * The ALARM event allows you to perform some actions
+     * after the command received a SIGALRM signal.
+     *
+     * @Event("Symfony\Component\Console\Event\ConsoleAlarmEvent")
+     */
+    public const ALARM = 'console.alarm';
```

Comment:
I'm wondering whether we should add an alias for the ConsoleAlarmEvent as `console.alarm`. Those aliases are needed for events that existed before our migration to PSR-14, but new events should not need them

Reply:
I'm fine either way, but as far as I can tell, currently all built-in events have an alias, maybe keep it for consistency?

Reply:
that's not true. Messenger events don't have aliases for instance.

Existing console events have aliases because they are old events (i.e. they already existed in Symfony 4)

Reply:
True, I missed those. In any case, if needed, I'll remove the alias.

Reply:
Done https://github.com/symfony/symfony/compare/33fb7b7a503113689e9ce3474e21b9e120865d39..5541f0095dfeeedab95cfba8b585d50cd90400bb